src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -139,6 +150,13 @@ const SearchFieldList = ({ onSearch }: Props) => {
   const [departments, setDepartments] = useState<string[]>([]);
   const [buildings, setBuildings] = useState<string[]>([]);
   const [halls, setHalls] = useState<string[]>([]);
+  const [userProvidedYear, setUserProvidedYear] = useState(new Date().getFullYear());
```

Comment:
This doesn't need to be a state variable, since it's never updated. You can make it a constant outside the component.

Reply:
Also, it's probably clearer to name it `currentYear`, since this variable is not actually user provided.

Reply:
Yes that is true, thank you! Just changed that